styled-components ^5.x.x support

I try to use styled-components in my project.

After installing (npm) react-static-plugin-styled-components I was expecting styled-components to be also installed into to node_modules - this did not happened.
After installing manually `npm i styled-components` I got this error:

npm ERR! peer dep missing: styled-components@^4.3.2, required by react-static-plugin-styled-components@7.3.0

I can fix it but it will be good to have support of latest version of styled-components if possible.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,7 @@
 ### Bugfix
 
 - Remove `@types/react-hot-loader` from TypeScript template ([#1485](https://github.com/react-static/react-static/pull/1485))
+- Expand `styled-components` peer dependency version range in `react-static-plugin-styled-components` to allow newer versions of styled-components to be used ([#1473](https://github.com/react-static/react-static/pull/1473))
 
 ## 7.4.1
 

--- a/packages/react-static-plugin-styled-components/README.md
+++ b/packages/react-static-plugin-styled-components/README.md
@@ -2,12 +2,16 @@
 
 A [React-Static](https://react-static.js.org) plugin that adds CSS-in-JS/SSR support for [styled-components](https://styled-components.com)
 
+## Prerequisites
+
+- styled-components 4+ (This has been tested with `4.3.2`)
+
 ## Installation
 
 In an existing react-static site run:
 
 ```bash
-$ yarn add react-static-plugin-styled-components
+$ yarn add react-static-plugin-styled-components styled-components
 ```
 
 Then add the plugin to your `static.config.js`:

--- a/packages/react-static-plugin-styled-components/package.json
+++ b/packages/react-static-plugin-styled-components/package.json
@@ -16,7 +16,7 @@
   },
   "peerDependencies": {
     "react": "^16.9.0",
-    "styled-components": "^4.3.2"
+    "styled-components": ">=4"
   },
   "devDependencies": {
     "@babel/cli": "^7.5.5",

